transports/sqs/configuration-options_visibility-timeout_sqs_[,7.3).partial.md

```diff
@@ -0,0 +1,11 @@
+## Message visibility
+
+The message visibility timeout in Amazon SQS defines how long a message remains hidden from other consumers after being received. If a handler takes longer to process a message than the configured visibility timeout, the message becomes visible again and may be reprocessed.
+
+Without a message visibility extension mechanism (supported in version 7.3 and higher), this can result in:
```

Comment:
```suggestion
Without a message visibility extension mechanism (supported in version 7.3 and above), this can result in:
```
I think we use above terminology, but I didn't check